types/json-rpc-error/index.d.ts

```diff
@@ -0,0 +1,30 @@
+// Type definitions for json-rpc-error 2.0
+// Project: https://github.com/claudijo/json-rpc-error
+// Definitions by: Leonid Logvinov <https://github.com/LogvinovLeon>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+declare class JsonRpcError extends Error {}
+
+declare namespace JsonRpcError {
+    class ParseError extends JsonRpcError {
+        constructor();
+    }
+    class InvalidRequest extends JsonRpcError {
+        constructor();
+    }
+    class InvalidParams extends JsonRpcError {
+        constructor();
+    }
+    class InternalError extends JsonRpcError {
+        constructor(err: Error | string);
```

Comment:
err needs to be optional as per https://github.com/claudijo/json-rpc-error#internal-error